next.config.js

```diff
@@ -1,8 +1,24 @@
+/** @type {import('next').NextConfig} */
 const config = {
   poweredByHeader: false,
   experimental: {
     outputStandalone: process.env.STANDALONE_BUILD === 'true',
   },
+  async rewrites() {
+    return {
+      beforeFiles: [
+        {
+          /** Rewrite for Elo voting */
```

Comment:
This is what tells Next to rewrite the routes to use Elo voting instead of the old voting system